src/js/ui/menu.js

```diff
@@ -304,6 +310,11 @@ Menu.prototype.selection = function(callback) {
   simply.impl.menuSelection();
 };
 
+Menu.prototype.setSelection = function(sectionIndex, itemIndex) {
+  this._selection = { sectionIndex: sectionIndex, itemIndex: itemIndex };
+  this._select();
+};
+
```

Comment:
Nice! Can you move this functionality into Menu.prototype.selection? It would overload `selection` to either be a getter or setter based on the first parameter.

It's a little more work implementation and documentation wise, but it falls in line with the general API of the other functions. It's a little odd since I think it'll be the first setter / async-getter.